llama-index-core/llama_index/core/indices/base.py

```diff
@@ -203,16 +203,17 @@ def insert_nodes(self, nodes: Sequence[BaseNode], **insert_kwargs: Any) -> None:
             self._insert(nodes, **insert_kwargs)
             self._storage_context.index_store.add_index_struct(self._index_struct)
 
-    def insert(self, document: Document, **insert_kwargs: Any) -> None:
+    def insert(self, document: Document, **kwargs: Any) -> None:
```

Comment:
nit: can we just leave the name as `insert_kwargs`

Reply:
OK, thanks for your reminder.